fhirstore/search/search_methods.py

```diff
@@ -54,16 +59,33 @@ def build_element_query(key, value):
         elif string_modifier == "below":
             element_query["simple_query_string"]["query"] = f"({value})*"
             element_query["simple_query_string"]["fields"] = [string_field]
+        elif string_modifier == "identifier":
+            element_query["simple_query_string"]["query"] = value
```

Comment:
nit: why is it `value` here and `f"{value}"` line 50?

Reply:
You're right, I fixed it